src/liballoc/collections/linked_list.rs

```diff
@@ -1843,3 +1843,15 @@ unsafe impl<T: Send> Send for IterMut<'_, T> {}
 
 #[stable(feature = "rust1", since = "1.0.0")]
 unsafe impl<T: Sync> Sync for IterMut<'_, T> {}
+
+#[unstable(feature = "linked_list_cursors", issue = "58533")]
+unsafe impl<T: Send> Send for Cursor<'_, T> {}
```

Comment:
```suggestion
unsafe impl<T: Sync> Send for Cursor<'_, T> {}
```
To match the bounds on `Iter`.

Reply:
Wow, i overlooked this! Thank you for point it out.